docs/joss/paper.md

```diff
@@ -0,0 +1,193 @@
+    orcid: 0000-0002-3920-4356
+    email: milank@mit.edu
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: "1, 2" # (Multiple affiliations must be quoted)
+
+  - name: Maximilian Gelbrecht
+    affiliation: "3,4"
+
+  - name: Daisuke Hotta
+    affiliation: "5,6"
+
+  - name: Justin Willmert
+    affiliation: 7
+
+  - name: Simone Silvestri
+    affiliation: 1
+
+  - name: Gregory L Wagner
+    affiliation: 1
+
+  - name: Alistair White
+    affiliation: "3,4"
+
+  - name: Sam Hatfield
+    affiliation: 8
+
+  - name: David Meyer
+    affiliation: 8
+
+  - name: Tom Kimpson
+    affiliation: "2,9"
+
+  - name: Navid C Constantinou
+    affiliation: 10
+
+  - name: Chris Hill
+    affiliation: 1
+    
+affiliations:
+ - name: Massachusetts Institute of Technology, Cambridge, MA, USA
+   index: 1
+ - name: University of Oxford, UK
+   index: 2
+ - name: Technical University Munich, Germany
+   index: 3
+ - name: Potsdam Institute for Climate Impact Research, Germany
+   index: 4
+ - name: Japan Meteorological Agency, Tsukuba, Japan
+   index: 5
+ - name: University of Reading, UK
+   index: 6
+ - name: University of Minnesota, Minneapolis, MN, USA
+   index: 7
+ - name: European Centre for Medium-Range Weather Forecasts, Reading, UK
+   index: 8
+ - name: University of Melbourne, Australia
+   index: 9
+ - name: Australian National University, Canberra, Australia
+   index: 10
+   
+date: 14 September 2023
+bibliography: paper.bib
+
+---
+
+
+# Summary
+
+SpeedyWeather.jl is a library to simulate and analyse the global atmospheric
+circulation on the sphere. It implements several 2D and 3D
+models to solve the primitive equations with and without humidity (\autoref{fig:primitive}),
+the shallow water equations (\autoref{fig:swm}), or the barotropic vorticity equations
+with spherical harmonics. Several simple parameterizations for unresolved physical processes
+such as precipitation or the boundary layer are implemented, and new ones can
+be externally defined and passed as an argument to the model constructor.
+SpeedyWeather.jl is an intermediate-complexity general circulation model [@Kucharski2013]
+and research playground with an (almost) everything-flexible attitude.
+It can be thought of as a conceptual reinvention of the Fortran SPEEDY model [@Molteni2003]
+in the Julia programming language [@Bezanson2017].
+
+SpeedyWeather.jl internally uses three sub-modules `SpeedyTransforms`, `RingGrids`, and
+`LowerTriangularMatrices` to perform spherical harmonic transforms and interpolations
+between various grids and the spectral space. `RingGrids` discretize the sphere
+on iso-latitude rings and the spectral space is defined by the `LowerTriangularMatrices`
+of the spherical harmonic coefficients. These three modules are independently usable
+and therefore make SpeedyWeather.jl, beyond its main purpose of simulating the weather,
+also a library for the analysis of gridded data on the sphere.
```

Comment:
If these three submodules are independently usable would it makes sense to have them as separate repositories in the SpeedyWeather organization?

Reply:
Yes, that is eventually the plan. However, I felt like this should wait a bit to make sure how these modules interact is a bit more established. Like I wanted to avoid that if we'd need to patch something in RingGrids, we have to wait for CI, release before having it also avaible in SpeedyWeather

Reply:
I see. Yeah perhaps single GPU is a good first goal. But also no need to mention anything in the paper then.